doc/source/Guidance-for-Reviewing-Code.rst

```diff
@@ -0,0 +1,108 @@
+    perspective, ensure that a subject matter expert is satisfied with
+    the changes.
+10. (Internal staff only:) Prior to moving the issue into the “Done”
+    column, notify the Product Owner that the issue has been completed,
+    and discuss the functionality implemented.
+
+**As a Reviewer:**
+------------------
+
+1. Issues should be assigned to a main reviewer. Adding multiple
+   reviewers to the pull request is encouraged to help share expertise.
+   Please ask for help assigning reviewers if you need it.
+
+2. All reviewers are encouraged to add comments to the pull request.
+
+3. Main reviewers should:
+
+   1. **Read the code and post in-line comments for suggested
+      alterations.**
+   2. **Ensure unit tests are run and pass.**
+   3. **Ensure command line interface acceptance tests run and pass.
+      These must be ran on the desktop using bin/improver tests as they
+      are not run by travis**
+   4. **The Acceptance Criteria defined within the associated issue has
+      been met.**
+   5. **The criteria within the Definition of Done has been satisfied.**
+   6. **Ensure their testing is documented on the issue.**
+
+4. Main reviewers should post comments to the pull request to show that
+   they have completed: i, ii, iii, iv, v, vi.
+
+5. Things to consider when reading through the code are:
+
+   1. **Naming conventions and coding style**
+
+      -  Does the code follow the expected protocols for formatting,
+         style and naming?
+      -  Do the names chosen make sense?
+
+   2. **Design**
+
+      -  How does the code fit in with overall IMPROVER design?
+      -  Is the code in the right place?
+      -  Is the code logical?
+      -  Could the new code have reused existing code?
+
+   3. **Readability and Maintainability**
+
+      -  Are the names used meaningful?
+      -  Are the functions understandable, with the use of the provided
+         docstrings and comments?
+      -  Are any warnings raised when running new unit tests.
+
+   4. **Functionality**
+
+      -  Does the code do what is supposed to?
+      -  Are errors handled appropriately?
+
+   5. **Test coverage**
+
+      -  Do the tests provided cover the expected situations?
+      -  Are the tests understandable?
+      -  Have edge cases been considered?
+
+6. If this is a first review, the developer should then move the issue
+   into “Second Review” and a second reviewer should ensure that the
+   criteria listed above have been met. The Scrum Master can be
+   consulted, if necessary.
+
+7. If this is a second review, the developer should contact the Product
+   Owner prior to moving the issue into the “Done” column. The Scrum
+   Master can be consulted, if necessary.
+
+**Before merging:**
+-------------------
+
+Always rerun Travis before merging a PR. Do this by: 1. At the bottom of
+the PR, click ``show all checks`` and then ``details`` next to the
+Travis check. 2. Click ``Restart build``
```

Comment:
1 and 2 here should be a bulleted list to make it clearer; it's become inline in the translation process.

Reply:
Fixed.